apps/src/templates/progress/GoogleClassroomShareButton.jsx

```diff
@@ -67,7 +61,7 @@ export default class GoogleClassroomShareButton extends React.PureComponent {
 
   componentDidMount() {
     this.renderButton();
-    this.resizeObserver.observe(this.buttonRef);
+    this.setState({buttonRendered: true});
```

Comment:
can you rename this prop to `mounted` since it no longer means that the button is rendered?

Reply:
Good catch! Done